src/styles/blocks/star.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+
+  &-star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &-star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} &-star:nth-child(-n + #{$starCount}) {
+      background-image: url(/images/star-active.svg);
```

Comment:
It's recommended to use relative paths for images. Use './images/star-active.svg' instead of '/images/star-active.svg'.